src/sentry/api/serializers/models/event.py

```diff
@@ -232,9 +232,9 @@ def serialize(self, obj, attrs, user):
                 received = None
 
         d = {
-            "id": six.text_type(obj.id),
```

Comment:
So this should be fine because, when serializing a snuba event we would already be using event id here. Is that right ?

Reply:
Yup